barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.test.cpp

```diff
@@ -443,10 +416,50 @@ template <typename Builder> class stdlib_bigfield : public testing::Test {
                 std::cout << "num gates per div = " << after - before << std::endl;
                 benchmark_info(Builder::NAME_STRING, "Bigfield", "DIV", "Gate Count", after - before);
             }
-            // uint256_t modulus{ Bn254FqParams::modulus_0,
-            //                    Bn254FqParams::modulus_1,
-            //                    Bn254FqParams::modulus_2,
-            //                    Bn254FqParams::modulus_3 };
+
+            fq expected = (inputs[0] / inputs[1]);
```

Comment:
I think you can just use uint256_t(c);

Reply:
Yes but just wanted to consistent with other (similar) tests.